src/pb_view.cpp

```diff
@@ -136,8 +136,9 @@ void pb_view::run(bool auto_download) {
 			if (idx != -1) {
 				dlstatus status = ctrl->downloads()[idx].status();
 				if (status == dlstatus::FINISHED || status == dlstatus::PLAYED || status == dlstatus::READY) {
-					ctrl->play_file(ctrl->downloads()[idx].filename());
-					ctrl->downloads()[idx].set_status(dlstatus::PLAYED);
+					if (ctrl->play_file(ctrl->downloads()[idx].filename())) {
```

Comment:
Right, I totally missed the fact that we should display a message here. Thanks for noticing!